### PR TITLE
Chore : fixed the backling issue, now it points to lbp instead of lbpmanager

### PR DESCRIPTION
**Pull request by codebuster22**

### CODE REVIEW

File: src/launches/launches.ts [hunk 1]
```diff
@@ -90,11 +91,22 @@ export class Launches {
   }
 
   gotoEtherscan(launch: ILaunch, event: Event): boolean {
-    Utils.goto(this.ethereumService.getEtherscanLink(launch.address));
+    if (launch.launchType === LaunchType.LBP ){
+      Utils.goto(this.ethereumService.getEtherscanLink(this.getLbpAddress(launch.address)));
```
Comment (dkent600) [inline src/launches/launches.ts:95]: should just be:  `Utils.goto(this.ethereumService.getEtherscanLink((launch as LbpManager).lbp.address));

(or some type cast like that)

File: src/launches/launches.ts [hunk 2]
```diff
@@ -90,11 +91,22 @@ export class Launches {
...
+    } else {
+      Utils.goto(this.ethereumService.getEtherscanLink(launch.address));
+    }
     event.stopPropagation();
     return false;
   }
 
+  getLbpAddress(lbpManager: Address): Address{
```
Comment (dkent600) [inline src/launches/launches.ts:103]: remove

File: src/launches/launches.ts [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import { EventAggregator } from "aurelia-event-aggregator";
-import { EthereumService } from "services/EthereumService";
+import { EthereumService, Address } from "services/EthereumService";
```
Comment (dkent600) [inline src/launches/launches.ts:2]: can remove Address?

**Review by dkent600 — CHANGES_REQUESTED** [review]

File: src/launches/launches.ts (hunk 2 above)
Comment (codebuster22) [reply src/launches/launches.ts]: removed

File: src/launches/launches.ts (hunk 3 above)
Comment (codebuster22) [reply src/launches/launches.ts]: removed

File: src/launches/launches.ts (hunk 1 above)
Comment (codebuster22) [reply src/launches/launches.ts]: implemented your suggestion

**Review by dkent600 — APPROVED** [review]

**Pull request merged**